contributed_definitions/NXtransmission.nxdl.xml

```diff
@@ -99,10 +100,10 @@ of instrument.-->
                      used to generate the result file(s) with measured data and metadata.
                 </doc>
             </field>
-            <field name="identifier" type="NX_CHAR">
+            <field name="identifier">
```

Comment:
Semantics-wise the class is the same but whitespace differences and differences whether type is mentioned explicitly I am still convinced it is useful to make as much as possible explicit to assist with understanding for beginners, no dealbreaker though as this class looks right now